app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -364,10 +370,38 @@ class BrowserFragment :
             owner = this,
             view = view
         )
-
+        setColorsForCustomTab(customTabConfig)
         setSitePermissions(view)
     }
 
+    /**
+     * Base on toolbar color set the same color for StatusBar and NavigationBar ,if the tab is custom.
+     */
+    private fun setColorsForCustomTab(customTabConfig: CustomTabConfig?) {
```

Comment:
Why do we need to do this? Does passing the `window` in the constructor of the custom tabs feature not work properly?

In general, when we try to fix bugs, we should consider:
1. Does it affect all our other browsers?
2. Do we want to change the behaviour of just _this particular_ browser or all our browsers?

Accordingly, we should decide if a change should be upstreamed to Android Components or if the feature can be written in a re-usable way so that our other browsers can also "compose" them into their app.

Reply:
I made the changes and I opened another pull request because of the commit author 
https://github.com/mozilla-mobile/focus-android/pull/6996

Reply:
I will closed this one

Reply:
@jonalmeida